src/InboundServer/index.js

```diff
@@ -18,53 +18,35 @@ const fs = require('fs');
 const path = require('path');
 const EventEmitter = require('events');
 
-const { WSO2Auth } = require('@mojaloop/sdk-standard-components');
-
 const Validate = require('../lib/validate');
 const router = require('../lib/router');
 const handlers = require('./handlers');
 const middlewares = require('./middlewares');
-const check = require('../lib/check');
 
 class InboundApi extends EventEmitter {
-    constructor(conf, logger, cache, validator) {
+    constructor(conf, logger, cache, validator, wso2) {
         super({ captureExceptions: true });
         this._conf = conf;
         this._cache = cache;
-        this._wso2 = {
-            auth: new WSO2Auth({
-                ...conf.wso2.auth,
-                logger,
-                tlsCreds: conf.inbound.tls.mutualTLS.enabled && conf.inbound.tls.creds,
-            }),
-            retryWso2AuthFailureTimes: conf.wso2.requestAuthFailureRetryTimes,
-        };
-        this._wso2.auth.on('error', (msg) => {
-            this.emit('error', 'WSO2 auth error in InboundApi', msg);
-        });
 
         if (conf.validateInboundJws) {
-            this._jwsVerificationKeys = conf.pm4mlEnabled ? conf.peerJWSKeys :  InboundApi._GetJwsKeys(conf.jwsVerificationKeysDirectory);
+            this._jwsVerificationKeys = conf.pm4mlEnabled ? conf.peerJWSKeys : InboundApi._GetJwsKeys(conf.jwsVerificationKeysDirectory);
```

Comment:
```suggestion
            // peerJWSKey is a special config option specifically for Payment Manager for Mojaloop
            // that is populated by a management api.
            // This map supersedes local keys that would be loaded in by jwsVerificationKeysDirectory.
            this._jwsVerificationKeys = conf.pm4mlEnabled ? conf.peerJWSKeys : InboundApi._GetJwsKeys(conf.jwsVerificationKeysDirectory);
```

Reply:
A short comment here or in config.js for peerJWSkeys would be appreciated since this variable doesn't appear anywhere else in this repo.